packages/astro/src/runtime/server/transition.ts

```diff
@@ -32,6 +32,7 @@ function toValidIdent(name: string): string {
 	if (/^[0-9]|^-[0-9]|^--/.test(name)) {
     throw new Error("Your transition:name is not a valid CSS identifier.");
 	}
+	// can't be a CSS keyword
 	if (['unset', 'initial', 'inherit', 'none' ].includes(name)) {
```

Comment:
Top! As mdn says, you can't define those. And it would definitively be a mistake to use them with `transition:name` in combination with `transition-animate` to define a new transition animation.

But you could use them as `view-transition-names` as they have a predefined meaning. At least `none` might come in handy if you want to exclude an element from the default animation.